docs/ROADMAP.md

```diff
@@ -1,3 +1,59 @@
+## Q1 2020 Roadmap
+
+### Overview
+
+- Migration: it will continue to be our focus, including filling up the details
+  to the migration guide and adding tooling to make migration easier.
+- Tackle issues labelled as `bug` or `developer experience`
+- Issue cleanup: take some time to triage open issues and possibly some smaller
+  enhancements that we never get the time to do.
+
+### Scope
+
+#### Migration
+
+- Migration guide https://github.com/strongloop/loopback-next/issues/453
+  - continue the work in Q4 2019 for migrating:
+    - boot scripts
+    - model mixins
+    - remoting hooks
+- Tooling:
+  - improve existing commands like `import-lb3-models`
+  - add new commands (e.g. import datasources).
+
+#### Authentication
+
+- Auth0 example: Create a more realistic example than
+  https://github.com/raymondfeng/loopback4-example-auth0.
+
+#### API Connect / LoopBack 4 Integration
+
+- Spike on getting APIs created from LB4 imported into APIC,
+  https://github.com/strongloop/loopback-next/issues/4115
+
```

Comment:
I am proposing to add the following section:

```markdown
#### Create models & REST APIs dynamically at runtime

- Dynamic binding/rebinding of controllers after app start #433
- How to build models, repositories and controllers dynamically at runtime #4296
- Docs for exposing REST API of a Model with no custom classes #2740 (stretch goal)
```

(See the spike https://github.com/strongloop/loopback-next/pull/4235).